src/tiles/item-stats.ts

```diff
@@ -0,0 +1,67 @@
+import { CSSResultGroup, html, LitElement } from 'lit';
```

Comment:
blocker: We want component names to be clear and accurate.

Since this component is part of both `account-tile` and `item-tile` it's not an "item" component, but a "tile" component.

Please rename to `tile-stats.ts`